src/math/unary_ops_test.ts

```diff
@@ -1309,6 +1309,18 @@ import {Array1D, Array2D, Scalar} from './ndarray';
       };
       expect(f).toThrowError();
     });
+
```

Comment:
Let's add tests for `Scalars` to check considering it is not supported yet.

Reply:
Since you are implementing it soon, I decided to not add an explicit unit test for that for now.